### DISCUSSION TITLE
Can you have two dimensional byte arrays in a table?

### DISCUSSION THREAD

**Discussion by randyK-HH** [dvidelabs/flatcc#246] [Q&A] [answered]
Can you have two dimensional byte arrays in a table?

I am aware we can have simple byte arrays in a table with variable length. IE:

```
table MyData
{
    my_data : [uint8];
}
```

Is it possible to add a dimension to "my_data" so that I can have an array of byte arrays (with variable length) ?

Something like:

```
table MyData
{
    my_data : [uint8:uint8];
}
```

or

```
table MyData
{
    my_data : [uint8][uint8];
}
```

Though I know that the 2D syntax examples (above) do not seem to compile.

**Comment by mikkelfj** [comment, marked as answer]
No, neither fixed length no variable length multidimensional support is possible in vectors. You need to introduce a vector of tables to achieve this, or, wing it with a vector of strings.

If you are aware of this being introduced in Googles flatc, but not in flatcc, please let know.
